src/lib.rs

```diff
@@ -25,11 +25,9 @@ use std::io::prelude::*;
 use std::io::{self, Error, ErrorKind, SeekFrom};
 use std::iter::repeat;
 use std::mem;
+use std::path::{Path, PathBuf};
 use std::str;
 
-#[cfg(feature = "nightly")]
```

Comment:
If possible, I would like to try to keep the changes to allow unpack to be used.  What I am trying to do is remove duplicated code from [libpcre-sys' build script](https://github.com/cadencemarseille/rust-pcre/blob/616d00a4075abc59ba571ee0a280068983891dd1/libpcre-sys/build.rs#L50).